libpod/image/image.go

```diff
@@ -156,6 +156,41 @@ func (ir *Runtime) New(ctx context.Context, name, signaturePolicyPath, authfile
 	return &newImage, nil
 }
 
+// NewFromLoad creates a new image object for images pulled from a tar archive (podman load)
+// This function is needed because it is possible for a tar archive to have multiple tags for one image
+func (ir *Runtime) NewFromLoad(ctx context.Context, name, signaturePolicyPath string, writer io.Writer) ([]*Image, error) {
```

Comment:
Can you rename this to `Load()` or `LoadFromArchive()`

Reply:
fixed